build-go.sh

```diff
@@ -130,7 +133,7 @@ function buildWithMatrix() {
 	# build each os/arch combo
 	while read line
 	do
-		doBuild $line $gobin $output
```

Comment:
i used it on line 130 because i declared it in that function. it wasnt passed into this one.